llvm/include/llvm/Support/PropertySetIO.h

```diff
@@ -158,6 +209,32 @@ class PropertySetRegistry {
   MapTy PropSetMap;
 };
 
+// Base64 encoder/decoder. Used to encode/decode byte array properties.
```

Comment:
I looked through those, they are not suitable for my purposes as is - significant additional work is needed to make those general purpose.

Reply:
> significant additional work is needed to make those general purpose

It should be easier than writing a new implementation from scratch. Right?
I expect that this will be required anyway when this code is upstreamed to llorg.
If you are going to address this separately, please add a TODO comment.

Reply:
> It should be easier than writing a new implementation from scratch. Right?
actually no. I'd need to grok code designed for different purpose to see what potential bugs it may present in my case.

I think the right direction here is to move this to a separate header/source inside the generic support library and then update other Base64 users (only 1 for now) to use this version instead of highly customized one, or maybe moving the customization to the generic library. I can add this kind of TODO, I can also move this to a separate source - is that OK?

Reply:
okay